fixtures/docstring/tests/bindings/test_docstring.kts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import uniffi.fixture.docstring.*;
```

Comment:
TBH I think a better option here might be to sprinkle a few docstrings in another fixture (eg, coveralls), with that other fixture *not* checking the docstring semantics, with this fixture trying to limit itself to just the docstring semantics. If every fixture had to test it's main point for existing *and* the union of all other capabilities just to ensure it didn't break something else things would be unmanageable quickly.

Eg, in this particular use-case, I think it sends a better signal to *not* have the test here at all (or a very very small skeleton) so the casual reader doesn't misunderstand what is being tested here.

(It's a shame it's not trivial to extract these, but I see no reason to block landing on a solution to this. It might be worth getting an issue on file after we land to help motivate some future contributor)

Reply:
I'm not exactly sure what is confusing you in this test. Just like the other tests, this test ensures that bindings code is generated correctly when docstrings are attached. This test *is* limiting itself to docstrings semantics. The test is not actually testing the functionality of other feature, merely the presence of them. Callback is not being invoked, errors are not being thrown. I don't see any other way to test that docstrings aren't breaking the syntax of generated bindings. I could add this docstrings to `coverall`, but I'm not sure I agree that making mega fixtures like coverall is the way to go. Overtime they will just continue to grow, and become more and more unmanagable.